clients/drcachesim/tools/schedule_stats.cpp

```diff
@@ -397,7 +397,7 @@ schedule_stats_t::print_counters(const counters_t &counters)
         std::cerr << ": ";
         auto it = counters.threads.begin();
         while (it != counters.threads.end()) {
-            std::cerr << *it;
+            std::cerr << it->workload_id << "." << it->tid;
```

Comment:
It may be better to format this as: W0.T123; seeing just 0.123 made me initially wonder why is it a floating point number.

Reply:
Done.